src/Adapter/Order/CommandHandler/IssuePartialRefundHandler.php

```diff
@@ -103,18 +98,16 @@ public function handle(IssuePartialRefundCommand $command)
 
             $orderDetail = new OrderDetail($orderDetailId);
 
-            if (empty($refund['amount'])) {
-                $refund['amount'] = $command->getTaxMethod() ?
+            if (null === $orderDetailRefund->getRefundAmount()) {
+                $productRefundAmount = $isTaxIncluded ?
                     $orderDetail->unit_price_tax_excl :
                     $orderDetail->unit_price_tax_incl;
-                $refund['amount'] *= $quantity;
+                $productRefundAmount *= $quantity;
             } else {
-                $refund['amount'] = $command->getTaxMethod() ?
-                    $taxCalculator->removeTaxes($refund['amount']) :
-                    $taxCalculator->addTaxes($refund['amount']);
+                $productRefundAmount = $orderDetailRefund->getRefundAmount();
```

Comment:
```suggestion
                $productRefundAmount = $orderDetailRefund->getRefundAmount() ?? (($isTaxIncluded ?                     $orderDetail->unit_price_tax_excl : $orderDetail->unit_price_tax_incl) * $quantity);
```

Reply:
Usually I like ternary assignation, but in this case it's a double ternary and it deals with a code/feature that is already complicated enough to grasp So I would prefer to keep the explicit notation, in order to focus on the expected behaviour

Reply:
:ok_hand: